Source/WebKit/UIProcess/glib/RendererBufferDescription.h

```diff
@@ -31,7 +31,11 @@
 namespace WebKit {
 
 struct RendererBufferDescription {
-    enum class Type : bool { DMABuf, SharedMemory };
+    enum class Type : uint8_t {
+        DMABuf,
+        SharedMemory,
+        AHardwareBuffer,
```

Comment:
Shouldn't this be OS(ANDROID) guarded?

Reply:
Could be, I was trying to avoid unneeded guards, but there is not much of a point of having all the rest with `OS(ANDROID)` and not this small bit. I'll add it before landing.